src/JustSaying/JustSaying.csproj

```diff
@@ -8,8 +8,10 @@
   <ItemGroup>
     <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.3.100.1" />
     <PackageReference Include="AWSSDK.SQS" Version="3.3.100.1" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
     <PackageReference Include="System.Text.Json" Version="4.6.0" />
+    <PackageReference Include="System.Threading.Channels" Version="4.7.0" />
```

Comment:
Would an older version such as `4.5.0` be sufficient to provide the same functionality so we have a wider compatibility range?

Reply:
I don't think an older version provides us with wider compatibility? This package on nuget supports any target framework we'd ever want to support.

Reply:
In fact, Channels 4.5.0 doesn't support `.NETCoreApp 3.0`

Reply:
Users of `netcoreapp3.0` would get an upgrade to the later version from the framework itself, so they'd get a newer version with optimisations.